src/pybind/mgr/dashboard/frontend/src/app/shared/components/custom-login-banner/custom-login-banner.component.scss

```diff
@@ -0,0 +1,4 @@
+p {
+  font-weight: bold;
+  padding: 15px;
+}
```

Comment:
you can make the scss file empty by utilizing the bootstrap classes.
```
<p class="p-3 font-weight-bold" 
     *ngIf="customText">{{customText}}</p>
```
(you can choose between `p-3` or `p-4` as the preferred padding)

I think this makes the downstream branding process easier.